src/System.Private.CoreLib/src/System/Activator.cs

```diff
@@ -77,7 +78,26 @@ public static T CreateInstance<T>()
         }
 
         [Intrinsic]
-        private extern static T CreateInstanceIntrinsic<T>();
+        private static T CreateInstanceIntrinsic<T>()
```

Comment:
(I validated that the intrinsic is still expanded in NUTC and we don't try to use the newly added body.)